### PR TITLE
Add UR Pstop Manager

### PR DESCRIPTION
**Pull request by MarqRazz**
This PR moves the `moveit_studio_ur_pstop_manager` package to this repo.

### CODE REVIEW

File: src/moveit_studio_ur_pstop_manager/include/moveit_studio_ur_pstop_manager/mock_ur_dashboard_client.hpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 PickNik Inc.
+// All rights reserved.
+//
+// Unauthorized copying of this code base via any medium is strictly prohibited.
+// Proprietary and confidential.
```
Comment (sea-bass) [inline src/moveit_studio_ur_pstop_manager/include/moveit_studio_ur_pstop_manager/mock_ur_dashboard_client.hpp:5]: Need to change all these license statements to the BSD 3-Clause in this repo :/

File: src/moveit_studio_ur_pstop_manager/package.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>moveit_studio_ur_pstop_manager</name>
+  <version>4.0.0</version>
+  <description>Provides a node to monitor the protective stop state of the UR5, and reset protective
+    stops when necessary.</description>
+  <maintainer email="cory.crean@picknik.ai">Cory Crean</maintainer>
+  <license>Proprietary</license>
```
Comment (sea-bass) [inline src/moveit_studio_ur_pstop_manager/package.xml:9]: Same here, need to change the maintainer to the generic "MoveIt Pro maintainer" and license to BSD 3-Clause. Basically whatever is in the other packages.

File: src/moveit_studio_ur_pstop_manager/include/moveit_studio_ur_pstop_manager/mock_ur_dashboard_client.hpp (hunk 1 above)
Comment (sea-bass) [reply src/moveit_studio_ur_pstop_manager/include/moveit_studio_ur_pstop_manager/mock_ur_dashboard_client.hpp:5]: This license statement didn't get updated.

File: src/moveit_studio_ur_pstop_manager/include/moveit_studio_ur_pstop_manager/protective_stop_manager_node.hpp [hunk 3]
```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 PickNik Inc.
+// All rights reserved.
+//
+// Unauthorized copying of this code base via any medium is strictly prohibited.
+// Proprietary and confidential.
```
Comment (sea-bass) [inline src/moveit_studio_ur_pstop_manager/include/moveit_studio_ur_pstop_manager/protective_stop_manager_node.hpp:5]: nor did this one

File: src/moveit_studio_ur_pstop_manager/package.xml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>moveit_studio_ur_pstop_manager</name>
+  <version>4.0.0</version>
+  <description>Provides a node to monitor the protective stop state of the UR5, and reset protective
+    stops when necessary.</description>
+  <maintainer email="cory.crean@picknik.ai">Cory Crean</maintainer>
```
Comment (sea-bass) [inline src/moveit_studio_ur_pstop_manager/package.xml:8]: maintainer also didn't get updated, just copy what's in the other packages here

File: src/moveit_studio_ur_pstop_manager/test/launch/test_reset_ur_pstop.test.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 PickNik Inc.
+# All rights reserved.
+#
+# Unauthorized copying of this code base via any medium is strictly prohibited.
+# Proprietary and confidential.
```
Comment (sea-bass) [inline src/moveit_studio_ur_pstop_manager/test/launch/test_reset_ur_pstop.test.py:7]: This license statement also didn't get updated yet

File: src/moveit_studio_ur_pstop_manager/test/test_reset_ur_pstop.cpp [hunk 6]
```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 PickNik Inc.
+// All rights reserved.
+//
+// Unauthorized copying of this code base via any medium is strictly prohibited.
+// Proprietary and confidential.
```
Comment (sea-bass) [inline src/moveit_studio_ur_pstop_manager/test/test_reset_ur_pstop.cpp:5]: and here

Comment (MarqRazz) [reply src/moveit_studio_ur_pstop_manager/test/test_reset_ur_pstop.cpp]: Sorry, I did this in a rush yesterday.

**Review by sea-bass — DISMISSED** [review]
Thank you!

Edit: Seems there is some copyright stuff in the tests that is still failing.

**Comment by sea-bass** [comment]
I think this will be fixed by changing

```
// Copyright (c) 2023 PickNik Inc.
// All rights reserved.
```

to what is in the other files:

```
// Copyright 2023 PickNik Inc.
```

**Pull request merged**